examples/todo-list/templates/index.html

```diff
@@ -0,0 +1,28 @@
+{% let request = request %}
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TODO List</title>
+</head>
+<body>
+<h1>TODO List</h1>
+<form id="todo-form" action="{{ flareon::reverse_str!(request, "add-todo") }}" method="post">
```

Comment:
Agreed, but I think sadly this will require forking `askama` library, or at least creating some wrapper proc macro. Forking `askama` to make it more convenient to use with flareon is on our roadmap, and I think it's out of scope for this PR, so I'd keep it as is for now.